app/assets/stylesheets/forms/_links.scss

```diff
@@ -21,8 +21,6 @@ input[type="submit"].not-button {
 
 	&:disabled {
 		opacity: 0.5;
+		cursor: initial;
 	}
```

Comment:
seen this used a couple times

might be worth doing this if this is the general disabled strategy:
```
@mixin disabled {
    opacity: .5;
    cursor: initial
}
```